xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -481,7 +657,7 @@ require(['jquery', 'xwiki-events-bridge'], function($) {
     var commentsNumber = $('.xwikicomment').size();
     if(commentsTab) {
       commentsTab.text("$services.localization.render('docextra.extranb', ['__number__'])".replace("__number__", commentsNumber));
-    };
+    }
     if($('#tmComment').length) {
       // All the sub-nodes of tmComment are added in a normalized form.
       $('#tmComment')[0].normalize();
```

Comment:
This is very hard to review.. since you moved code from one place to another it's hard to see what exactly was changed. Would have been better to try to minimize the diff, especially since this is a large file that serves an important feature...